pkg/apis/servicecatalog/v1beta1/types.go

```diff
@@ -1057,7 +1057,7 @@ type ServiceBindingSpec struct {
 
 	// List of transformations that should be applied to the credentials
 	// associated with the ServiceBinding before they are inserted into the Secret.
-	SecretTransform []SecretTransform `json:"secretTransform,omitempty"`
```

Comment:
as this is a breaking change, did it ever get released? We are ok dropping the old secretTransform object ? I am.

Reply:
Agreed, I think it's just fine to drop/rename at this point.